src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -262,6 +262,10 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             if (isset($this->aliases[$id])) {
                 $id = $this->aliases[$id];
             }
+            if (isset($this->privates[$id]) && !$this instanceof ContainerBuilder) {
```

Comment:
why the check on ContainerBuilder ? this is useless (`$this->privates` will be empty for them anyway)